components/PulseButton.tsx

```diff
@@ -0,0 +1,97 @@
+import React, {useState, useEffect} from "react";
+import { Animated, StyleSheet, Easing } from "react-native";
+import { MaterialIcons } from "@expo/vector-icons";
+
+import { colors } from "./theme";
+
+type Props = {
```

Comment:
The convention is to use an interface, start the name with an 'I' and name it for it's purpose. 
e.g.
```
interface IPulseButtonProps {
    shouldPulse: boolean;
    position: string;
    onPress: () => void;
};
```